Description: Define AC_ACVERSION for newer autoconf
 Patch upstream aclocal.m4 to define AC_ACVERSION for newer autoconf
 so that the upstream M4 macros work.
 .
Author: Dave Beckett <dajobe@debian.org>
Last-Update: 2013-02-13

--- byacc-20121003.orig/aclocal.m4
+++ byacc-20121003/aclocal.m4
@@ -35,6 +35,7 @@ dnl
 dnl $1 = version to compare against
 dnl $2 = code to use if AC_ACVERSION is at least as high as $1.
 dnl $3 = code to use if AC_ACVERSION is older than $1.
+ifdef([AC_ACVERSION], ,[m4_copy([m4_PACKAGE_VERSION],[AC_ACVERSION])])
 define([CF_ACVERSION_CHECK],
 [
 ifdef([m4_version_compare],
